README.md

```diff
@@ -905,11 +921,14 @@ params do
   requires :avatar, type: File
 end
 post '/' do
-  # Parameter will be wrapped using Hashie:
-  params.avatar.filename # => 'avatar.png'
-  params.avatar.type     # => 'image/png'
-  params.avatar.tempfile # => #<File>
+  # Parameter will be wrapped using HashWithIndifferentAccess:
+  params[:avatar][:filename] # => 'avatar.png'
+  params['avatar']['avatar'] # => 'image/png'
+  params[:avatar][:tempfile] # => #<File>
 end
+
+`params` hash keys  can accesed with `""` or `:`
+`:avatar` and `"avatar"` are considered to be the same.
```

Comment:
The old syntax should work by default, so put that back. And is this true when using `Hash`? What is it storing?

Ruby programmers know the difference so maybe say "as a string" or "as a symbol".